### PR TITLE
Unit tests for DELETE persons/:id and DELETE encounters/:id

### PR DESCRIPTION
**Pull request by sher812**
Added route tests for DELETE persons/:d and DELETE encounters/:id endpoints.

image

Contribution: @UoA-sjer215 and @sher812

### CODE REVIEW

**Review by butterplup — APPROVED** [review]
LGTM ✅

File: backend/src/routes/__test__/encounter.route.test.ts [hunk 1]
```diff
@@ -426,6 +429,436 @@ describe('PUT /encounters/:id ', () => {
...
 
+// Delete Encounters Endpoint tests
+
+// Delete Encounter 200
+describe('DELETE /encounter/:id', () => {
+    it('Successfully deletes single encounter with single person: ', async () => {
+        // Get Authentication ID for User
+        const auth_id = await testUtils.getAuthIdFromToken(token);
+
+        // Create Person
```
Comment (conrad-scherb) [inline backend/src/routes/__test__/encounter.route.test.ts:440]: There's a lot of repeated code between these test cases. I'd strongly recommend creating some functions to create people & encounters for your tests in order to remove a bunch of this repeated code at the start of each test case.

Comment (butterplup) [reply backend/src/routes/__test__/encounter.route.test.ts:440]: conrad what are you doing here?

**Review by UoA-sjer215 — APPROVED** [review]
lgtm

**Pull request merged**